src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,153 @@
+* `az webapp hybrid-connection add`: Improve help/error message and unblock Linux (#18843)
+* `az webapp config access-restriction remove`: Fix #18947 issue removing service endpoint rules (#18986)
+* : Fix #17424: `az appservice plan show`: Provide correct exit status (#18994)
+
+**ARM**
+
+* `az what-if`: Fix output formatting (#18721)
+* `az bicep uninstall`: Add new command to uninstall bicep (#18744)
+* `az bicep build`: Fix an issue where running with --stdout doesn't print any output (#18744)
+* `az provider register`: Add deprecate info for `--accept-term` (#18739)
+* `az lock create/delete`: Add examples for operating different levels of locks (#18890)
+* `az deployment group/sub/mg/tenant create`: Add --what-if parameter for invoking What-If with the deployment create commands. (#18924)
+* `az deployment group/sub/mg/tenant create`: Add --proceed-if-no-change parameter to skip confirmation when --confirm-with-what-if is set and there's no changes in What-If results. (#18924)
+* Bump api-version from 2020-10-01 to 2021-04-01 (#18923)
+* `az ts create`: Make parameter `--template-file` support bicep file (#18888)
+* Add example for creating site extension to web app (#18935)
+* `az ts export`: Fix the issue that export template specs with no linked templates failed (#18928)
+
+**PolicyInsights**
+
+* Migrate to track 2 SDK (#18740)
+
+**Backup**
+
+* `az backup vault`: Add support for Customer Managed Keys(CMK) (#18733)
+* `az backup restore restore-disks`: Add MSI usage in IaaS VM Restore (#18961)
+
+**CDN**
+
+* `az cdn endpoint rule`: Add OriginGroupOverride action support (#18711)
+
+**Compute **
+
+* Fix #18633: az sig image-version create does not support mixing disks, snapshots, and vhd (#18741)
+* Upgrade package version to fix securityProfile issue in `az vmss update` (#18788)
+* `az vm boot-diagnostics get-boot-log`: Fix crash when getting boot diagnostics log (#18830)
+* `az vm list-skus`: Fix the issue that it can't query the SKU which with partially zones available (#18939)
+* `az vm auto-shutdown`: Fix the issue that `--webhook` is required when `--email` is passed in (#18958)
+* `az vm create`: Support creating VM from shared gallery image (#19037)
+* `az vm secret add`: Add note to use Azure Key Vault VM extension instead in help (#19045)
+
+**Container**
+
+* `az container exec`: Fix and improve terminal experience (#18909)
+
+**DataBoxEdge**
+
+* Migrate databoxedge to track2 SDK (#18678)
+
+**DMS**
+
+* `az dms project create/az dms project task create`: Removing MySQL projects/tasks for online migrations since they are no longer supported. (#18709)
+
+**IoT**
+
+* `az iot hub create/update`: Added checks to prevent bad file-upload identity parameters when hub doesn't have identity (#18966)
+* `az iot hub create/update`: Added `--fileupload-notification-lock-duration` parameter (#18966)
+* `az iot hub create/update`: Deprecated `fileupload-storage-container-uri` parameter (#18966)
```

Comment:
```suggestion
* `az iot hub create/update`: Add checks to prevent bad file-upload identity parameters when hub doesn't have identity (#18966)
* `az iot hub create/update`: Add `--fileupload-notification-lock-duration` parameter (#18966)
* `az iot hub create/update`: Deprecate `fileupload-storage-container-uri` parameter (#18966)
```